server/Server.ts

```diff
@@ -34,6 +34,7 @@ const REGISTRY_REDWOOD_HOSTNAME_PATTERN =
 const REGISTRY_HAMBACH_HOSTNAME_PATTERN =
   /hambach--regen-registry\.netlify\.app$/;
 const REGISTRY_V4_HOSTNAME_PATTERN = /v4--regen-registry\.netlify\.app$/;
+const MAIN_PREVIEW_HOSTNAME_PATTERN = /\d+--regen-registry\.netlify\.app$/;
```

Comment:
to account for non digit characters in the preview hash
```suggestion
const MAIN_PREVIEW_HOSTNAME_PATTERN = /[a-z0-9]+--regen-registry\.netlify\.app$/;
```

Reply:
Seems like this part has only numbers but let's play it safe you're right.

Reply:
in https://639aee43101e366250794f27--regen-registry.netlify.app/, 639aee43101e366250794f27 does have some letters